internal/test/wmcb/wmcb_test.go

```diff
@@ -67,6 +67,7 @@ func TestWMCBUnit(t *testing.T) {
 	assert.NoError(t, err, "error reading stdout from the remote Windows VM")
 	os.Stdout = stdout
 	assert.NotContains(t, string(out), "FAIL")
+	assert.Contains(t, string(out), "PASS")
```

Comment:
Im not sure, but wont this not catch the case where some tests pass and then a test or helper panics?

Reply:
Have you tried reading the exit code from WinRM.Run()? Wouldnt that return 0 on a successful `go test`?